src/index.html

```diff
@@ -11,7 +11,183 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <div class="form-widget">
+          <label for="surname">Surname:</label>
+          <input
+          name="surname"
+          id="surname"
+          type="text"
+          autocomplete="off"
+          required
+          >
+        </div>
+
+        <div class="form-widget">
+          <label for="name">Name:</label>
+          <input
+          name="name"
+          id="name"
+          type="text"
+          autocomplete="off"
+          required
+          >
+        </div>
+
+        <div class="form-widget">
+          <label for="age">How old are You?</label>
+          <input
+          name="age"
+          id="age"
+          type="number"
+          min="1"
+          max="100"
+          value="12"
+          required
+          >
+        </div>
+
+        <div class="form-widget">
+          <label for="full_date_of_birth">Full date of birth:</label>
+          <input
+          name="full_date_of_birth"
+          id="full_date_of_birth"
```

Comment:
Fix in all cases

[checklist p4](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)